framework/include/utils/InputParameters.h

```diff
@@ -1872,20 +1887,21 @@ template <>
 void InputParameters::addPrivateParam<MultiMooseEnum>(const std::string & /*name*/);
 
 template <>
-void InputParameters::addDeprecatedParam<MooseEnum>(const std::string & name,
-                                                    const std::string & doc_string,
-                                                    const std::string & deprecation_message);
+void InputParameters::addDeprecatedParam<MooseEnum>(const std::string & /*name*/,
+                                                    const std::string & /*doc_string*/,
+                                                    const std::string & /*deprecation_message*/);
 
 template <>
-void InputParameters::addDeprecatedParam<MultiMooseEnum>(const std::string & name,
-                                                         const std::string & doc_string,
-                                                         const std::string & deprecation_message);
+void
+InputParameters::addDeprecatedParam<MultiMooseEnum>(const std::string & /*name*/,
+                                                    const std::string & /*doc_string*/,
+                                                    const std::string & /*deprecation_message*/);
 
 template <>
 void InputParameters::addDeprecatedParam<std::vector<MooseEnum>>(
-    const std::string & name,
-    const std::string & doc_string,
-    const std::string & deprecation_message);
+    const std::string & /*name*/,
+    const std::string & /*doc_string*/,
+    const std::string & /*deprecation_message*/);
```

Comment:
This isn't a big deal, but is there a reason for commenting out the names of these arguments? I don't think it makes any difference whether they're commented out, does it?

Reply:
No it doesn't from a functional standpoint. This was a code standards and consistency adjustment, given that the other methods did comment the arguments out. I was nearby and made this tweak when I noticed it.